app/src/main/java/com/example/myfridgehome/dto/SetStoredFoods.kt

```diff
@@ -3,11 +3,7 @@ package com.example.myfridgehome.dto
 import android.app.Notification
 import android.app.Notification.MessagingStyle.Message
 import android.util.JsonWriter
-import java.io.IOException
-import java.io.OutputStream
-import java.io.OutputStreamWriter
-import java.io.StringWriter
-
+import java.io.*
```

Comment:
Add all java.io tools since you are using them.

Reply:
Using * in imports is considered bad practice, mostly for documentation reasons.   
1) It's hard to determine where certain classes came from, if all imports are *
2) You can have collisions, like java.sql.Date and java.util.Date
3) Nearly all modern IDEs will organize imports for you, so there's no extra effort in importing each class separately.